libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2268,6 +2268,51 @@ MAV_RESULT GCS_MAVLINK::handle_command_set_message_interval(const mavlink_comman
     return MAV_RESULT_FAILED;
 }
 
+MAV_RESULT GCS_MAVLINK::handle_command_get_message_interval(const mavlink_command_long_t &packet)
+{
+    if (comm_get_txspace(chan) < PAYLOAD_SIZE(chan, MESSAGE_INTERVAL) + PAYLOAD_SIZE(chan, COMMAND_ACK)) {
+        return MAV_RESULT_TEMPORARILY_REJECTED;
+    }
+
+    const uint32_t mavlink_id = (uint32_t)packet.param1;
+    if (mavlink_id >= 2 << 15) {
+        // response packet limits range this works against!
+        mavlink_msg_message_interval_send(chan, mavlink_id, 0); // not available
+        return MAV_RESULT_ACCEPTED;
+    }
+
+    const ap_message id = mavlink_id_to_ap_message_id(mavlink_id);
+    if (id == MSG_LAST) {
+        gcs().send_text(MAV_SEVERITY_INFO, "No ap_message for mavlink id (%u)", mavlink_id);
```

Comment:
I wonder if it makes sense to send a text message here.  It's a bit unlikely that it is a human that is sending the get-message-interval and we're already sending a response that a robot would understand.